urbanopt/geojson-modelica-translator
# #252 feat: cooling district system

#### Any background context you want to provide?
GMT is transitioning to a more modular design for modelica components.

#### What does this PR accomplish?
Allows users of GMT to model district cooling systems. This is accomplished by adding:
- python code for modeling central cooling plants
- modelica code for coupling cooling plants to the network
- small fixes to other systems

#### How should this be manually tested?
Run test_district_cooling_system.py, load the resulting model into dymola and verify things look correct. Note that this test uses two loads (their icons will be overlapped in dymola)

#### What are the relevant tickets?
Addresses #252 

#### Screenshots (if appropriate)
<img width="496" alt="Screen Shot 2020-12-15 at 9 20 44 AM" src="https://user-images.githubusercontent.com/18518728/102241997-d39a3200-3eb6-11eb-9e16-b59620b5ac69.png">
